mlflow/pytorch/__init__.py

```diff
@@ -69,26 +69,25 @@ def get_default_conda_env():
         :caption: Output
 
         conda env {'name': 'mlflow-env',
-                   'channels': ['defaults', 'conda-forge', 'pytorch'],
-                   'dependencies': ['python=3.7.5', 'pytorch=1.5.1',
-                                    'torchvision=0.6.1',
-                                    'pip', {'pip': ['mlflow', 'cloudpickle==1.6.0']}]}
+                   'channels': ['conda-forge'],
+                   'dependencies': ['python=3.7.5',
+                                    {'pip': ['pytorch=1.5.1',
+                                             'torchvision=0.6.1',
+                                             'mlflow',
+                                             'cloudpickle==1.6.0']}]}
```

Comment:
```suggestion
                                    {'pip': ['pytorch==1.5.1',
                                             'torchvision==0.6.1',
                                             'mlflow',
                                             'cloudpickle==1.6.0']}]}
```

Relate to https://github.com/mlflow/mlflow/pull/4393/files#r640267418